Data/ByteString/Lazy.hs

```diff
@@ -894,6 +895,20 @@ index cs0 i         = index' cs0 i
               index' cs (n - fromIntegral (S.length c))
           | otherwise       = S.unsafeIndex c (fromIntegral n)
 
+-- | /O(c)/ 'ByteString' index, starting from 0, that returns 'Nothing' if:
+--
+-- > 0 <= n < length bs
```

Comment:
I believe it returns `Just`, if this condition holds?

Reply:
:facepalm: